vochain/indexer/queries/account.sql

```diff
@@ -0,0 +1,20 @@
+-- name: UpsertAccount :execresult
+INSERT INTO accounts(
+  account, balance, nonce
+) VALUES (
+    ?, ?, ?
+) ON CONFLICT(account)
+  DO UPDATE SET
+    balance = excluded.balance,
+    nonce = excluded.nonce
+;
```

Comment:
this should just be REPLACE INTO; see queries/votes.sql for an example

Reply:
Have you benchmarked your approach to actually be noticeably faster, or is this explicitly documented upstream? as far as I can tell both methods will work fine, and REPLACE is shorter and simpler, so that's what I have been using

Reply:
Let's apply our current standard which is currently REPLACE, and let's check in the future if it is worth it to change everything to UPSERT or not. Thanks @mariajdab for taking the time to consider a potential faster alternative.

Reply:
Sure, sounds good! :raised_hands: It would be worth confirming with the benchmark test in the future